### PR TITLE
PIC-1254 Update Prison Categories

### PR DESCRIPTION
**Pull request by theDustRoom**

### CODE REVIEW

File: app/services/stubs/StubPrisonerApi.java [hunk 1]
```diff
@@ -45,6 +46,7 @@ public StubPrisonerApi() {
                 .builder()
                 .firstName("Sam")
                 .surname("Jones")
+                .dateOfBirth("1980-12-01")
```
Comment (theDustRoom) [inline app/services/stubs/StubPrisonerApi.java:49]: You can manually test by changing this dateOfBirth so the offender is under 21

File: app/controllers/ParoleParom1ReportController.java [hunk 2]
```diff
@@ -113,11 +121,13 @@ public ParoleParom1ReportController(HttpExecutionContext ec,
 
     private Map<String, String> storeCustodyData(Map<String, String> params, String bearerToken, Optional<PrisonerApi.Offender> maybeOffender, Optional<PrisonerCategoryApi.Category> maybeCategory) {
         val nomsNumber = params.get("prisonerDetailsNomisNumber");
+
         params.putAll(maybeOffender
                 .map(offender -> ImmutableMap.<String, String>builder()
                         .put("prisonerDetailsPrisonInstitution", offender.getInstitution().getDescription())
                         .put("prisonerDetailsPrisonNumber", offender.getMostRecentPrisonerNumber())
                         .put("prisonerDetailsPrisonersFullName", offender.displayName())
+                        .put("isYoungAdult", String.valueOf(calculateAge(Optional.ofNullable(offender.getDateOfBirth()).orElse("1970-01-01"), systemUTC()) < 21))
```
Comment (theDustRoom) [inline app/controllers/ParoleParom1ReportController.java:130]: This ensures that if dateOfBirth is somehow not present, we will treat them as over 21

**Review by andymarke — APPROVED** [review]
LGTM

**Review by katie-cawthorne — APPROVED** [review]
LGTM 👍

**Pull request merged**